package.json

```diff
@@ -6,11 +6,14 @@
   "description": "A UI application for interfacing with Sia",
   "license": "MIT",
   "devDependencies": {
+    "babel-eslint": "^6.0.4",
     "chai": "^3.4.1",
     "chai-as-promised": "^5.1.0",
     "electron-mocha": "^0.7.0",
     "electron-packager": "^7.0.2",
     "electron-prebuilt-compile": "^1.2.0",
+    "eslint": "^2.11.0",
+    "eslint-plugin-react": "^5.1.1",
     "ink-docstrap": "^1.0.3",
     "jsdoc": "^3.4.0",
     "jshint": "^2.9.1-rc1"
```

Comment:
can we get ride of some of these dependencies now? Specifically `chai`, `chai-as-promised`, `electron-mocha`, `ink-docstrap`, `jsdoc`, and `jshint`?